src/test/components/LocalTrack.test.js

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+
+import {
+  changeSelectedThread,
+  hideLocalTrack,
+} from '../../actions/profile-view';
+import LocalTrack from '../../components/timeline/LocalTrack';
+import {
+  getLocalTracks,
+  getRightClickedTrack,
+} from '../../reducers/profile-view';
+import { getSelectedThreadIndex } from '../../reducers/url-state';
+import { getProfileWithNiceTracks } from '../fixtures/profiles/tracks';
+import { storeWithProfile } from '../fixtures/stores';
+
+const LEFT_CLICK = 0;
+const RIGHT_CLICK = 2;
+
+describe('timeline/LocalTrack', function() {
+  /**
+   *  getProfileWithNiceTracks() looks like: [
+   *    'show [thread GeckoMain process]',
+   *    'show [thread GeckoMain tab]',
+   *    '  - show [thread DOM Worker]',         <- use this local track.
+   *    '  - show [thread Style]',
+   *  ]
+   */
+  function setup() {
+    const pid = 222;
```

Comment:
is there a way to retrieve this number ? We also could have a comment to indicate why we use 222

Reply:
It's a somewhat arbitrary identifier, so it's probably as effective as looking the value through some other index. I'll add an explanatory comment.

```
    // In getProfileWithNiceTracks, the two pids are 111 and 222 for the
    // "GeckoMain process" and "GeckoMain tab" respectively. Use 222 since it has
    // local tracks.
```